src/routes/users.route.ts

```diff
@@ -19,6 +19,7 @@ class UsersRoute implements Routes {
     this.router.post(`${this.path}`, validationMiddleware(CreateUserDto, 'body'), this.usersController.createUser);
     this.router.put(`${this.path}/:id(\\d+)`, validationMiddleware(CreateUserDto, 'body', true), this.usersController.updateUser);
     this.router.delete(`${this.path}/:id(\\d+)`, this.usersController.deleteUser);
+    this.router.put(`${this.path}/manageGroups/:id(\\d+)`, this.usersController.manageGroups);
```

Comment:
based on the openapi specfication is recomended to use kbab-case as path name, please change for this one: 

`this.router.put('${this.path}/manage-groups/:id(\\d+)', this.usersController.manageGroups);`

@ben-at-soul-refiner